### ISSUE TITLE
Aggregated data collected does not work

### ISSUE THREAD

**Issue by BluLupo** [MatteoJoliveau/convomark#32]
<a href="https://imgur.com/U59UH1o"><img src="https://i.imgur.com/U59UH1o.png" title="source: imgur.com" /></a>
The site https://analytics.matteojoliveau.com/index.php?module=CoreHome&action=index&idSite=1&period=day&date=yesterday#?idSite=1&period=day&date=year&category=Dashesterdayboard_Dashboard&subcategory=1 
which shows the collected data : https://convomark.matteojoliveau.com/privacy

does not work

**Comment by MatteoJoliveau** [comment]
Yeah that Matomo instance is no longer active. I should remove the link from the bot's dashboard
